<ui version="4.0" >
 <class>PluginManagerDialog</class>
 <widget class="QDialog" name="PluginManagerDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Plugins</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QListView" name="pluginsList" />
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <widget class="QPushButton" name="loadButton" >
           <property name="text" >
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="unloadButton" >
           <property name="text" >
            <string>Unload</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
